contrib/charts/dragonfly/templates/_pod.tpl

```diff
@@ -86,7 +86,7 @@ containers:
       {{- toYaml . | trim | nindent 6 }}
     {{- end }}
     {{- include "dragonfly.volumemounts" . | trim | nindent 4 }}
-    {{- if .Values.passwordFromSecret.enable -}}
+    {{- if .Values.passwordFromSecret.enable }}
```

Comment:
Can you run the golden chart rendering command and also include those changes?

https://github.com/dragonflydb/dragonfly/blob/main/CONTRIBUTING.md#rendering-helm-golden-files